docs/pages/index.mdx

```diff
@@ -40,6 +39,23 @@ slug: wallets
   </Button>
 </div>
 
+<style>{`
+  :root {
+    --react-quickstart-btn-bg: var(--accent-2, #ECF3FF);
+    --react-quickstart-btn-color: black;
+  }
```

Comment:
`--react-quickstart-btn-bg: var(--accent-2, #ECF3FF);`
Ya this doesn't make sense - it's trying to set a new variable using... an existing variable. So why wouldn't we use the original variable haha

Reply:
`--react-quickstart-btn-color: black;`
This line is the source of the problem. It's basically the same as it was before. It's hard-coding the text color as `black` all the time which is not what we want. Fern's CSS natively handles text colors for us so we should not overwrite it.